sdk/keyvault/azure-security-keyvault-keys/src/test/java/com/azure/security/keyvault/keys/cryptography/CryptographyClientTest.java

```diff
@@ -204,6 +244,70 @@ public void signVerifyEc(HttpClient httpClient, CryptographyServiceVersion servi
                 assertTrue(verifyStatus);
             }
         }
+    }
+
+    @Test
+    public void signVerifyEc() throws NoSuchAlgorithmException, InvalidAlgorithmParameterException {
+        Map<KeyCurveName, SignatureAlgorithm> curveToSignature = new HashMap<>();
+        curveToSignature.put(KeyCurveName.P_256, SignatureAlgorithm.ES256);
+        curveToSignature.put(KeyCurveName.P_384, SignatureAlgorithm.ES384);
+        curveToSignature.put(KeyCurveName.P_521, SignatureAlgorithm.ES512);
+        curveToSignature.put(KeyCurveName.P_256K, SignatureAlgorithm.ES256K);
+
+        Map<KeyCurveName, String> curveToSpec = new HashMap<>();
+        curveToSpec.put(KeyCurveName.P_256, "secp256r1");
+        curveToSpec.put(KeyCurveName.P_384, "secp384r1");
+        curveToSpec.put(KeyCurveName.P_521, "secp521r1");
+        curveToSpec.put(KeyCurveName.P_256K, "secp256k1");
```

Comment:
On OSX, I wouldn't be surprised if secp256k1 curves didn't work. .NET uses openssl and OSX doesn't support that curve. Just FYI.

Reply:
...it also doesn't seem to handle explicit curves, not that I suspect you have that problem now. But if you ever support explicit curves, you likely will. I only have them in .NET because of the new `CertificateClient.DownloadCertificate` method group I added to return `System.Security.Cryptography.X509Certificates.X509Certificate2`.